### DISCUSSION TITLE
Blog Not To Render When Date is Later

### DISCUSSION THREAD

**Discussion by brycemcwhirter** [facebook/docusaurus#10487] [Q&A]
Hey Devs, 

I'm currently working on my blog blog.brycemcwhirter.com and I was curious if there was any existing functionality within docusaurus to not render a blog post if the date comes after the current date?

Thanks! :)

**Comment by slorber** [comment]
You can use front matter:
- `draft: true`: blog post is only visible in dev)
- `unlisted: true`: blog post is browsable in prod but "hidden" from sidebar, sitemap, SEO no index...)

You can also use the v3.1 `parseFrontMatter` hook to implement custom logic and add `draft` or `unlisted` flags based on other front matter like `date`

https://docusaurus.io/blog/releases/3.1#parsefrontmatter-hook

Now keep in mind that Docusaurus is a static site generator. Your blog post will not magically appear once time passes, and if your blog post is now today/yesterday, you'll probably need to rebuild your site to ensure Docusaurus generates the appropriate static content for the new blog post. 

IMHO it's better to be explicit and use `draft` / `unlisted` directly. If you need to publish a post at a given time, you can prepare a PR and use a scheduling tool to merge it at the right time. Apparently there are GitHub apps for that: https://github.com/marketplace/pr-scheduler

**Reply by brycemcwhirter** [reply]
Hey @slorber, 

Thank you so much! That PR Scheduler method should work great!
